test/phpunit/PhysicalobjectCsvImporterTest.php

```diff
@@ -0,0 +1,564 @@
+<?php
+
+use org\bovigo\vfs\vfsStream;
+
+class PhysicalObjectCsvImporterTest extends \PHPUnit\Framework\TestCase
+{
+  protected $csvHeader;
+  protected $csvData;
+  protected $daos;
```

Comment:
Oops, I seem to have declared $daos and then never used it.  I'll delete before merging.